### PR TITLE
Update GitHub links

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
A successful PR build will test many of these lnks.
Other links should be checked in a code review.

### CODE REVIEW

File: android/docker/Dockerfile [hunk 1]
```diff
@@ -72,7 +72,7 @@ RUN mkdir "$HIFI_BASE" && \
     mkdir "$HIFI_ANDROID_PRECOMPILED"
 
 # Download the repo
-RUN git clone https://github.com/kasenvr/project-athena.git
+RUN git clone https://github.com/vircadia/vircadia.git
 
 WORKDIR /home/gha/project-athena
```
Comment (HifiExperiments) [inline android/docker/Dockerfile:77]: some of the other project-athena references here also should be vircadia now

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-a96ef14.tar_fe738063.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-a96ef14_fe738063.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-a96ef14.tar_fe738063.gz

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-a96ef14_fe738063.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-a96ef14_fe738063.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-45004eb.tar_5a4210a0.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-45004eb.tar_5a4210a0.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-45004eb_5a4210a0.sh

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-45004eb_5a4210a0.dmg

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Quest-Alpha-PR908-45004eb_5a4210a0.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-45004eb_5a4210a0.apk

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-45004eb_5a4210a0.exe

**Review by HifiExperiments — APPROVED** [review]

**Review by JulianGro — APPROVED** [review]
I don't know how the files in cmake/ports/ work so I cannot approve those, but I haven't found any issues.

**Review by digisomni — APPROVED** [review]
Looks like everything is building (on GHA) correctly. :)

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-19c28c0.tar_086d4601.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-19c28c0_086d4601.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-19c28c0.tar_086d4601.gz

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-19c28c0_086d4601.dmg

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-19c28c0_086d4601.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Quest-Alpha-PR908-19c28c0_086d4601.apk

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR908/Vircadia-Alpha-PR908-19c28c0_086d4601.exe

**Pull request merged**